client/src/app/components/date-picker/date-picker.component.ts

```diff
@@ -13,6 +14,7 @@ export class DatePickerComponent implements OnInit, OnChanges {
     @Input() public pickedDate: Date;
     public date: FormControl = new FormControl(this.pickedDate);
     public time: NgbTimeStruct;
+    private dateTime:any = moment();
```

Comment:
You can try `moment.Moment` type instead of `any`.

Initialize it with `moment().startOf('day')` this way you will have time set to `0`

Reply:
Added type and startOf('day')